store/search/src/main/scala/org/apache/spark/rpc/Master.scala

```diff
@@ -69,20 +69,32 @@ class Master(sparkConf: SparkConf, port: Int) {
   }
 
   /** start service and listen on port passed in constructor */
-  def startService(): Unit = {
+  def startService(message: MasterStatus): Unit = {
```

Comment:
instead of adding this parameter, you can add a isStarted method in Master, and use it in SparkCarbonStore

Reply:
ok, done.